src/Menu.js

```diff
@@ -29,6 +40,7 @@ const Menu = ({
       selected={selected}
       className={className}
       onSelect={onSelect}
+      getTabIndex={getTabIndex}
```

Comment:
dont think this is the correct approach. should only pass a number/string. passing a function complicates the component interface. given this is not an event

Reply:
I have an idea let's see what you think...

Reply:
@lwhiteley what do you think?